libusb/os/windows_winusb.c

```diff
@@ -2581,7 +2590,10 @@ static int winusbx_claim_interface(int sub_api, struct libusb_device_handle *dev
 	struct libusb_context *ctx = HANDLE_CTX(dev_handle);
 	struct winusb_device_handle_priv *handle_priv = get_winusb_device_handle_priv(dev_handle);
 	struct winusb_device_priv *priv = usbi_get_device_priv(dev_handle->dev);
-	bool is_using_usbccgp = (priv->apib->id == USB_API_COMPOSITE);
+
+	//bool is_using_usbccgp = (priv->apib->id == USB_API_COMPOSITE);
```

Comment:
Please do not leave commented out code. If this line is no longer needed then remove it entirely.